enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationLimitService.java

```diff
@@ -107,4 +119,25 @@ public void checkIdentityVerificationLimit(OwnerId ownerId) throws PowerAuthClie
             throw new OnboardingProcessLimitException("Max failed attempts reached for identity verification");
         }
     }
+
+    /**
+     * Check the limit for maximum number of document uploads.
+     * @param ownerId Owner identifier.
+     * @throws IdentityVerificationLimitException Thrown in case document upload limit is reached.
+     * @throws RemoteCommunicationException Thrown when communication with PowerAuth server fails.
+     * @throws IdentityVerificationException Thrown in case identity verification is invalid.
+     * @throws OnboardingProcessLimitException Thrown when maximum failed attempts for identity verification have been reached.
+     */
+    public void checkDocumentUploadLimit(OwnerId ownerId, IdentityVerificationEntity identityVerification) throws IdentityVerificationLimitException, RemoteCommunicationException, IdentityVerificationException, OnboardingProcessLimitException {
+        List<DocumentVerificationEntity> documentVerificationsFailed = documentVerificationRepository.findAllDocumentVerifications(identityVerification, DocumentStatus.ALL_UNSUCCESSFUL);
```

Comment:
I see a little contrast between naming `failed` and `ALL_UNSUCCESSFUL`. Maybe we the constant `DocumentStatus.ALL_UNSUCCESSFUL` could be renamed to `ALL_FAILED` to keep consistent.

Reply:
Yes, we can change that.